### DISCUSSION TITLE
umi3.5，开启webpack5，组件与组件之间必须循环依赖，如何配置？或者不检查这个错误？

### DISCUSSION THREAD

**Discussion by susanred** [umijs/umi#8627] [General]
我们公司正在从旧webpack4脚手架到umi3升级，原脚手架并没有此问题，也没有做特别的设置，但在umi中，会提示此错误，阻止程序正常启动

报错信息

![image](https://user-images.githubusercontent.com/17443650/179887390-4e4b3ad3-f8c6-4b16-a320-0f0d7bbf0166.png)

**Comment by fz6m** [comment]
> 组件与组件之间必须循环依赖

js 中不能循环依赖，要把公共部分抽出来。

**Reply by susanred** [reply]
谢谢，但是更想知道是哪个导致的

**Comment by susanred** [comment]
最后解决了
在页面使用绝对路径引入文件，而非相对路径
